setups/node/action.yaml

```diff
@@ -14,10 +14,13 @@ runs:
     - name: Cache PNPM modules
       uses: actions/cache@v3
       with:
-        path: ~/.local/share/pnpm/store/v3
+        path: |
+          ~/.local/share/pnpm/store/v3
+          ~/.pnpm-store
+          **/node_modules
```

Comment:
do we need to add the `node_modules` here?
i found in their docs that's not recommended

reference: https://github.com/actions/cache/blob/main/examples.md#node---npm

Reply:
That's a good point, but shouldn't be an issue since the cache key is hardly tied to the cache key, according to the docs issues can only arrise when using different versions of node
![CleanShot 2024-07-29 at 17 59 10@2x](https://github.com/user-attachments/assets/e21bb836-6257-4322-8e71-35998bd3159b)